AppController/djinn.rb

```diff
@@ -3028,9 +2950,7 @@ def update_firewall()
       all_ips.join(', '))
 
     # Re-run the filewall script here since we just wrote the all_ips file
-    if FIREWALL_IS_ON
```

Comment:
absolutely right, no need for it in this PR. I was just working around that area and I removed it since it was not run-time configurable. I will re-add it for now.